Isosurfacing_3/include/CGAL/Dual_contouring_3.h

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) 2022 INRIA Sophia-Antipolis (France).
+// All rights reserved.
+//
+// This file is part of CGAL (www.cgal.org).
+//
+// $URL$
+// $Id$
+// SPDX-License-Identifier: GPL-3.0-or-later OR LicenseRef-Commercial
+//
+// Author(s)     : Julian Stahl
+//                 Daniel Zint
+
+#ifndef CGAL_DUAL_CONTOURING_3_H
+#define CGAL_DUAL_CONTOURING_3_H
+
+#include <CGAL/license/Isosurfacing_3.h>
+
+#include <CGAL/Isosurfacing_3/internal/Cell_type.h>
+#include <CGAL/Isosurfacing_3/internal/Dual_contouring_internal.h>
+#include <CGAL/tags.h>
+
+namespace CGAL {
+namespace Isosurfacing {
+
+/**
+ * \ingroup PkgIsosurfacing3Ref
+ *
+ * \brief Creates an indexed face set that represents an isosurface using the Dual Contouring algorithm.
+ *
+ * \details
+ *
+ * \tparam ConcurrencyTag determines if the algorithm is executed sequentially or in parallel.
+ *
+ * \tparam Domain_ must be a model of `IsosurfacingDomain`.
+ *
+ * \tparam PointRange is a model of the concept `RandomAccessContainer` and `BackInsertionSequence` whose value type can
+ * be constructed from the point type of the polygon mesh.
+ *
+ * \tparam PolygonRange a model of the concept
+ * `RandomAccessContainer` and `BackInsertionSequence` whose value type is itself a model of the concepts
+ * `RandomAccessContainer` and `BackInsertionSequence` whose value type is `std::size_t`.
+ *
+ * \tparam Positioning is a functor containing the operator() that takes `domain`, `iso_value`, `cell`, and `position`
+ * as input and returns a boolean that is true if the isosurface intersects the cell.
+ *
+ * \param domain the domain providing input data and its topology
+ * \param iso_value value of the isosurface
```

Comment:
As we use "isosurface"  (and should fix some occurrences of  "iso-surface") we should use "isovalue".